package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "3.31.3",
+  "version": "3.31.4",
   "engines": {
-    "node": "^8.0.0 || ^10.13.0"
+    "node": "^8.0.0 || ^10.13.0 || ^11.0.0 || ^12.0.0"
```

Comment:
Do we want to support 11 since it's not LTS? We don't support 7 or 9 😏

Reply:
Doneso